tests/acceptance/controllers/register-test.js

```diff
@@ -0,0 +1,68 @@
+import { module, test } from 'qunit';
+import { setupApplicationTest } from 'ember-qunit';
+import { visit, fillIn, click } from '@ember/test-helpers';
+
+module('Acceptance | Controllers | register', function(hooks) {
+  setupApplicationTest(hooks);
+  test('Should render Register with strong value on password meter', async function(assert) {
+    assert.expect(1);
+    await visit('/');
+
+    await fillIn('#password', 'Testing@234?:234');
+    assert.equal(
+      this.element.querySelectorAll('.strong').length,
+      6,
+      'There are 6 divs with strong class'
+    );
+  });
+  test('Should render Register with medium value on password meter', async function(assert) {
+    await visit('/');
+
+    await fillIn('#password', 'Testing@234?:');
+    assert.equal(
+      this.element.querySelectorAll('.medium').length,
+      4,
+      'There are 4 divs with medium class'
+    );
+  });
+  test('Should render Register with weak value on password meter', async function(assert) {
+    assert.expect(1);
+    await visit('/');
+
+    await fillIn('#password', 'Testing');
+    assert.equal(
+      this.element.querySelectorAll('.weak').length,
+      1,
+      'There are 1 div with weak class'
+    );
```

Comment:
I would personally write this test to assert:
- `statusText` was shown correctly
- strength meter displays correctly
- button is disabled or not disabled correctly

Acceptance tests are sometimes more useful than unit tests because they test the interaction, not just a straightforward and trivial cause and effect